src/assets/stylesheets/shared/base.sass

```
@@ -40,3 +38,9 @@ p
   margin:
     bottom: $m-sm
     top: 0
+
+.brand-icon
+  filter: invert(58%) sepia(52%) saturate(5269%) hue-rotate(214deg) brightness(95%) contrast(96%)
```

This is INSANE :D But please consider this solution :)
```suggestion
  fill: $brand-primary
```